job-app/src/components/popup/Popup.css

```diff
@@ -0,0 +1,37 @@
+.popup {
+    z-index: 1;
+    width: 100%;
+    height: 100%;
+    background-color: rgb(80,80,80);
+    background-color: rgba(80,80,80,0.5);
+    position: fixed;
+    left: 0;
+    top: 0;
+    overflow: auto;
+}
+
+.popup-body {
+    width: 500px;
+    height: 500px;
```

Comment:
Does the width/height sizing also need to follow 100% or 30vw/vh? Or is px fine

Reply:
So the idea behind the flexbox approach is to have a fixed size (in this case 500 x 500, although % or vw is also usable) and have the flex fillers fill up the rest of the space. So I think this part is correct but let's chat in the other comment